workflows/transcriptomics/goseq/README.md

```diff
@@ -0,0 +1,29 @@
+# Goseq GO-KEGG Enrichment Analysis Workflow
+
+## Inputs dataset
+
+The workflow need the following inputs:
+- The DEG file:
+    - A tabular file with first column the gene symbol and second column a boolean value whether the gene is DEG or not. 
+- The gene length file:
+    - A tabular file with first column the gene symbol and second column the gene length of the genes.
+- The KEGG file:
+    - A tabular file with first column the Pathway ID and second column the Pathway name like: 
+        -   ```
+            ID  Name
+            01100   Metabolic pathways - mmus
+            01200   Carbon metabolism - mmus 
+            ```
+ 
+## Processing
+
+- The workflow will do a simple enrichment analysis with taking into account the gene length
+- The output will be 3 files `GO table`, `Top ontology plot` and `DE genes in each category` for CC, BP, and MF ontologies and `KEGG table` and `DE genes in each KEGG Pathways`
+
+### Warning
+
+- This workflow uses mm10 reference genome
```

Comment:
Can you please make that configurable ? You can insert a simple workflow parameter and select "Attempt restriction base on connections"

Reply:
Would be good to also do it with the Select Gene ID format parameter.